network-api/networkapi/buyersguide/templates/about/why_we_made.html

```diff
@@ -1,73 +1,119 @@
+      goal is to help you shop smart—and safe—for products that connect to the internet.
+      {% endblocktrans %}
     </p>
-  
+
     <p class="body">
-      Three years ago when we first started <em>*Privacy Not Included</em>, we didn’t know if people would be interested in a
-      guide about the privacy and security of connected toys, gadgets, and smart home products. Turns out, they were. And
-      it wasn’t just people who were interested. We discovered companies were too. We’re happy to see both consumers and
-      companies increasingly value connected products that are safe, secure, and private.
+      {% blocktrans trimmed %}
+      Three years ago when we first started <em>*Privacy Not Included</em>, we didn’t know if people would be interested
+      in a guide about the privacy and security of connected toys, gadgets, and smart home products. Turns out, they
+      were. And it wasn’t just people who were interested. We discovered companies were too. We’re happy to see both
+      consumers and companies increasingly value connected products that are safe, secure, and private.
+      {% endblocktrans %}
     </p>
-  
+
     <p class="body">
+      {% blocktrans trimmed %}
       What can you expect in <em>*Privacy Not Included</em> this year?
+      {% endblocktrans %}
     </p>
   </div>
 
   <section aria-labelledby="minimum-security">
-    <p id="minimum-security"><b>Our Minimum Security Standards</b></p>
+    <p id="minimum-security"><b>{% trans "Our Minimum Security Standards" %}</b></p>
     <p class="body">
-      We realize people want to <em>just know</em> which products are safe and which aren’t. We are Mozilla—not a consumer product review company—so we won’t say “Buy this, don’t buy that.” Instead, we used our technical expertise to create a set of <a href="https://foundation.mozilla.org/en/privacynotincluded/about/meets-minimum-security-standards">Minimum Security Standards</a> we think all products should meet in order to be sold in stores. Those standards include using encryption, automatic security updates, requiring strong passwords, having a system to manage vulnerabilities, and having an accessible privacy policy.  Look for the “Meets Our Minimum Security Standards” badge on products in the guide. This year we’ve also added a star rating near the top of each product page so you can glance at a product and see just how well it does on our Minimum Security Standards.
+      {% blocktrans with url="https://foundation.mozilla.org/privacynotincluded/about/meets-minimum-security-standards" trimmed %}
+      We realize people want to <em>just know</em> which products are safe and which aren’t. We are Mozilla—not a
+      consumer product review company—so we won’t say “Buy this, don’t buy that.” Instead, we used our technical
+      expertise to create a set of <a href={{url}}>Minimum Security Standards</a> we think all products should meet in
```

Comment:
Here but also in many other places, should this be `<a href="{{ ... }}">` so that the quotes are preserved?

Reply:
D’oh, how did I miss them **all**? :) Fixed!